sdk/search/Azure.Search.Documents/tests/FieldBuilderTests.cs

```diff
@@ -611,7 +618,7 @@ private class ModelWithUnsupportedEnumerableType
             public string ID { get; set; }
 
             [SimpleField(IsFilterable = true)]
-            public IEnumerable<byte> Buffer { get; set; }
+            public IEnumerable<char> Buffer { get; set; }
```

Comment:
Any reason this changed from byte to char?

Reply:
Yes, this model is for unsupported EnumerableType. Now that we support byte, I updated the type which is unsupported.